Automattic/wp-calypso
# Emails: Add names and logos to In-Depth Comparison page

This pull request adds email provider names and logos to the `In-Depth Comparison` page:

![image](https://user-images.githubusercontent.com/594356/148099829-48096efe-2c78-4ba7-aac7-c4cc2f00f682.png)

It also moves all types to a dedicated file.

#### Testing instructions

Note you will have to append `?flags=emails/in-depth-comparison` to the url of the page (and reload it) if you're testing those changes on a live branch as this feature has not been enabled on this environment yet:

1. Run `git checkout update/in-depth-comparison-page-with-names-and-logos` and start your server, or access a [live branch](https://calypso.live/?image=registry.a8c.com/calypso/app:build-23884)
2. Log into a WordPress.com account with a domain but no email
3. Open the [`Emails` page](http://calypso.localhost:3000/email)
4. Click the `Add Email` button of that domain to access the `Email Comparison` page
5. Click the `See how they compare` link to access the `In-Depth Comparison` page
6. Assert that the page looks like in the screenshot above
